telemetry/cli.py

```diff
@@ -52,6 +52,10 @@ def log_boot_logs(server, in_args):
         "pytest_failures": 0,
         "pytest_skipped": 0,
         "pytest_tests": 0,
+        "matlab_errors": 0,
```

Comment:
Maybe its good also to update these for complete support/reference to the new fields, however these are optionals.

https://github.com/sdgtt/telemetry/blob/4322215a7e6a4e764b00f2a8bc453fa52b294853/telemetry/searches.py#L37

Tests
https://github.com/sdgtt/telemetry/blob/4322215a7e6a4e764b00f2a8bc453fa52b294853/tests/test_elastic.py#L207
https://github.com/sdgtt/telemetry/blob/4322215a7e6a4e764b00f2a8bc453fa52b294853/tests/test_elastic.py#L252